docs/guides/unity-catalog-azure.md

```diff
@@ -85,72 +87,18 @@ provider "databricks" {
 }
 ```
 
-## Configure Azure objects
-
-The first step is to create the required Azure objects:
-
-- An Azure storage account, which is the default storage location for managed tables in Unity Catalog. Please use a dedicated account for each metastore.
-- A Databricks Access Connector that provides Unity Catalog permissions to access and manage data in the storage account.
-
-```hcl
-resource "azurerm_databricks_access_connector" "unity" {
-  name                = "${local.prefix}-databricks-mi"
-  resource_group_name = data.azurerm_resource_group.this.name
-  location            = data.azurerm_resource_group.this.location
-  identity {
-    type = "SystemAssigned"
-  }
-}
-
-resource "azurerm_storage_account" "unity_catalog" {
-  name                     = "${local.prefix}storage"
-  resource_group_name      = data.azurerm_resource_group.this.name
-  location                 = data.azurerm_resource_group.this.location
-  tags                     = data.azurerm_resource_group.this.tags
-  account_tier             = "Standard"
-  account_replication_type = "GRS"
-  is_hns_enabled           = true
-}
-
-resource "azurerm_storage_container" "unity_catalog" {
-  name                  = "${local.prefix}-container"
-  storage_account_name  = azurerm_storage_account.unity_catalog.name
-  container_access_type = "private"
-}
-
-resource "azurerm_role_assignment" "example" {
-  scope                = azurerm_storage_account.unity_catalog.id
-  role_definition_name = "Storage Blob Data Contributor"
-  principal_id         = azurerm_databricks_access_connector.unity.identity[0].principal_id
-}
-```
-
 ## Create a Unity Catalog metastore and link it to workspaces
```

Comment:
If we remove storage root - should we put a note somewhere that all catalogs must have it then?